editor/project_manager/project_dialog.cpp

```diff
@@ -552,6 +553,21 @@ void ProjectDialog::ok_pressed() {
 		fa_icon->store_string(get_default_project_icon());
 
 		EditorVCSInterface::create_vcs_metadata_files(EditorVCSInterface::VCSMetadata(vcs_metadata_selection->get_selected()), path);
+
+		if (generate_editor_config_file->is_pressed()) {
+			const String editor_config_path = path.path_join(".editorconfig");
+			Ref<FileAccess> f = FileAccess::open(editor_config_path, FileAccess::WRITE);
+			if (f.is_null()) {
+				ERR_FAIL_MSG("Couldn't create .editorconfig in project path.");
```

Comment:
May be we should use `ERR_PRINT` instead? `.editorconfig` isn't so critical and `EditorVCSInterface::create_vcs_metadata_files()` result is also not checked.